codingup/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I added this.
But in future, please don't ever include `node_modules`, some generated code or secret credentials in the git repository.

Reply:
Why is there an empty `package.json` file included? Why is it empty?